Configuration/PyReleaseValidation/python/relval_production.py

```diff
@@ -25,6 +25,7 @@
 workflows[1040] =  ['',['RunZeroBias2017F','TIER0RAWSIPIXELCAL','ALCASPLITSIPIXELCAL','ALCAHARVDSIPIXELCAL']]
 workflows[1040.1] =  ['',['RunExpressPhy2017F','TIER0EXPSIPIXELCAL','ALCASPLITSIPIXELCAL','ALCAHARVDSIPIXELCAL']]
 
```

Comment:
I didn't want to be picky but can we remove the space line?

Reply:
Hi @chayanit 
As this WF is not related to the pixel calibration (only shares the same input stream) I wanted to emphasise it through the extra space. Maybe another possibility would be to change its float-like number to reflect this (but none of the previous reviewers raised this until now)?

Reply:
OK then we should move to different workflow ID. I would propose it to be 1041.0 (random pick but keep it makes sense)

Reply:
Thanks!
For my personal information, is there any well-defined guideline to pick up such numbers?

Reply:
Unfortunately no. It's also not clear to me what the rules are but there are some consistency for the decimal workflows. 

If anyone knows. please feel free to comment.

Reply:
1041 should be good. It will be your chain with ExpressPhy2017F input.

Reply:
Changed in 282611e, and left the whitespace to isolate from pixel calibration (another PCL component should appear soon, as discussed at [the last AlCaDB meeting](https://indico.cern.ch/event/1026590/contributions/4310538/).